src/Model.php

```diff
@@ -65,7 +67,19 @@
 	 */
 	public $exists = false;
 
-	// TODO implement timestamps
+	/**
+	 * The name of the "created at" column.
+	 *
+	 * @var string
+	 */
+	const CREATED_AT = 'create_time';
```

Comment:
I'm not sure I get why this is used. Just because there are inconsistencies in the database tables for column names?

Reply:
Also, how are the column aliases loaded initially? Do we keep them stored somewhere and load them for every table?

Reply:
I saw that the uses of column aliases are hardcoded into the table. If we are going to do this, it would be nice to do it automatically. Therefore a decision has to be bate regarding where to we enforce the aliases. Do we keep the Model `attributes` with the original column name, or with the alias? My 2 cents would be to keep it always with the original column name, and look for the alias every time the user interacts with the model. But, if this is too inefficient, then we can enforce it at the query level.

Reply:
Regarding the TimeStamp topic, would it be better to have a flag that tells the Model that it should be touched on update?